mingus/containers/note.py

```diff
@@ -272,7 +274,7 @@ def __int__(self):
                 res += 1
             elif n == "b":
                 res -= 1
-        return res
+        return int(res)
```

Comment:
It's not clear why this `int` should be needed. Did you ever get a non-int here? If so, maybe there's another bug to uncover.

Reply:
Sorry also not sure why I added this - I'm guessing it was from a type bug but I'm afraid I can't recall